### PR TITLE
Added more SC deploy scenario tests for transaction process status functionality 

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added more SC deploy scenario testsf or transaction process status functionality

### CODE REVIEW

File: process/testdata/finishedFailedSCDeployWithTransfer.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      "sourceShard": 4294967295,
+      "destinationShard": 2,
+      "status": "success",
+      "operation": "transfer",
+      "fee": "0",
+      "callType": "directCall",
+      "options": 0
+    }
+  ]
+}
```
Comment (miiu96) [inline process/testdata/finishedFailedSCDeployWithTransfer.json:125]: empty line at the end of the file

File: process/testdata/finishedOKSCDeployWithTransfer.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      "receiver": "erd1adfmxhyczrl2t97yx92v5nywqyse0c7qh4xs0p4artg2utnu90pspgvqty",
+      "sender": "erd1adfmxhyczrl2t97yx92v5nywqyse0c7qh4xs0p4artg2utnu90pspgvqty",
+      "status": "success",
+      "operation": "transfer",
+      "fee": "0",
+      "callType": "directCall",
+      "options": 0
+    }
+  ]
+}
```
Comment (miiu96) [inline process/testdata/finishedOKSCDeployWithTransfer.json:73]: empty line at the end of the file

File: process/testdata/finishedFailedSCDeployWithTransfer.json (hunk 1 above)
Comment (iulianpascalau) [reply process/testdata/finishedFailedSCDeployWithTransfer.json]: done

File: process/testdata/finishedOKSCDeployWithTransfer.json (hunk 2 above)
Comment (iulianpascalau) [reply process/testdata/finishedOKSCDeployWithTransfer.json]: done

**Review by miiu96 — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**